cmd/promlts/sidecar.go

```diff
@@ -19,51 +19,45 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
-	"github.com/oklog/oklog/pkg/group"
+	"github.com/improbable-eng/promlts/pkg/okgroup"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"gopkg.in/alecthomas/kingpin.v2"
 	yaml "gopkg.in/yaml.v2"
 )
 
-func registerSidecar(m map[string]runFunc, app *kingpin.Application, name string) {
+func registerSidecar(m map[string]setupFunc, app *kingpin.Application, name string) {
 	cmd := app.Command(name, "sidecar for Prometheus server")
 
-	apiAddr := cmd.Flag("api-address", "listen address for the store API").
-		Default(":19090").String()
+	apiAddr := cmd.Flag("sidecar.address", "listen address for the store API").
```

Comment:
When running `promlts sidecar ...` the default flag namespace is already kind of defined as `sidecar` no? Prefixing again makes the command more noisy, no?